crates/rspack_macros/src/rspack_version.rs

```diff
@@ -1,5 +1,5 @@
 pub fn rspack_version() -> String {
-  let re = regex::Regex::new(r#""version": ?"([0-9\.]+)""#).expect("should create regex");
+  let re = regex::Regex::new(r#""version": ?"([0-9a-zA-Z\.-]+)""#).expect("should create regex");
```

Comment:
this is kind of too hacky, even formatter will kill this

Reply:
why not using json parse?

Reply:
Our package.json is in a standard json format, so there should be no problem in theory.